### PR TITLE
Revise workflow for stalled objects

### PR DESCRIPTION
**Pull request by georglauterbach**
# Description

~~Follow up on #2300. Sorry for the extra work.~~

Revises the workflow for stale issues **and** PRs. One can now add the label `bot/ignore` to issues and PRs to indicate that the stale action should ignore this object. Furthermore, configuration for PRs was added explictly.

## Type of change

- [x] Improvement (non-breaking change that does improve existing functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (README.md or the documentation under `docs/`)
- [x] If necessary I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: .github/workflows/close_stale_issues.yml [hunk 1]
```diff
@@ -13,6 +13,7 @@ jobs:
         with:
           repo-token: ${{ secrets.GITHUB_TOKEN }}
           exempt-issue-labels: "issue/not-stale"
+          exempt-pr-labels: "pr/not-stale"
```
Comment (wernerfred) [inline .github/workflows/close_stale_issues.yml:16]:
```suggestion
          exempt-pr-labels: "bot/ignore"
```

**Review by casperklein — CHANGES_REQUESTED** [review]
I wasn't aware, that PRs are also considered by stale bot, as we have no explicit config for PRs.

The workflow file and title should be renamed, to reflect also PRs.
Also the close message, tags, etc.  should be streamlined. ATM for PRs the label `stale` is used instead of `meta/stale` like in issues.

**Review by wernerfred — CHANGES_REQUESTED** [review]
Maybe i am a bit late to the party as #2300 got already merged but i would highly prefer a solution like suggested (`bot/ignore` or variations like `stalebot/ignore`) over the now introduced `[issue|pr]/not-stale`.

Imho we should avoid making statements on the actual state of an issue or pr (what `not-stale` implies as we cannot guarantee it - its state could change in future or other PRs or might be totally different depending of the individual working on this - but only maintainers can edit the labels not the contributor) but rather highlight that certain automation does not apply here.

**Comment by georglauterbach** [comment]
I will take care.

File: .github/workflows/close_stale_issues.yml (hunk 1 above)
Comment (georglauterbach) [reply .github/workflows/close_stale_issues.yml]: 👍🏼

**Review by casperklein — APPROVED** [review]
LGTM. The only thing to consider is maybe, to write out "pull requests" instead of PR.

**Review by wernerfred — APPROVED** [review]
Thanks

**Comment by georglauterbach** [comment]
If "PR" turns out to be confusing in the future, I will change it :)

**Comment by polarathene** [comment]
Might not matter much, especially since this got merged already, but `bot/ignore` implies any bot should ignore it rather than something like `stale-bot/ignore`.

**Comment by georglauterbach** [comment]
As we only have one bot at the moment, all is well. If we implement a new bot, we can change this too.

**Pull request merged**